cmd/juju/machine/remove.go

```diff
@@ -237,3 +213,47 @@ func (c *removeCommand) Run(ctx *cmd.Context) error {
 	}
 	return nil
 }
+
+func logRemovedMachine(ctx *cmd.Context, result params.DestroyMachineResult, keepInstance bool) error {
+	if result.Error != nil {
+		return errors.Errorf("removing machine failed: %s", result.Error)
+	}
+	for _, destroyContainerResult := range result.Info.DestroyedContainers {
+		err := logRemovedMachine(ctx, destroyContainerResult, keepInstance)
+		if err != nil {
+			return err
```

Comment:
One container failing doesn't mean that subsequent containers failed as well. Log the error and keep going.